Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/BackLighting.azsli

```diff
@@ -49,26 +52,35 @@ float3 GetBackLighting(Surface surface, LightingData lightingData, float3 lightI
             // https://colinbarrebrisebois.com/2011/03/07/gdc-2011-approximating-translucency-for-a-fast-cheap-and-convincing-subsurface-scattering-look/
 
             {
-                thickness = max(shadowRatio, surface.transmission.thickness);
+                thickness = max(transmissionDistance, surface.transmission.thickness);
                 float transmittance = pow( saturate( dot( lightingData.dirToCamera, -normalize( dirToLight + surface.normal * transmissionParams.z ) ) ), transmissionParams.y ) * transmissionParams.w;
                 float lamberAttenuation = exp(-thickness * transmissionParams.x) * saturate(1.0 - thickness);
                 result = transmittance * lamberAttenuation * lightIntensity;
             }
             break;
 
         case TransmissionMode::ThinObject:
-            // Thin object mode, using thin-film assumption proposed by Jimenez J. et al, 2010, "Real-Time Realistic Skin Translucency"
+            // Thin object mode, based on Jimenez J. et al, 2010, "Real-Time Realistic Skin Translucency"
             // http://www.iryoku.com/translucency/downloads/Real-Time-Realistic-Skin-Translucency.pdf
-
-            float litRatio = 1.0 - shadowRatio;
-            if (litRatio)
+            
             {
-                const float thickness = surface.transmission.thickness * transmissionParams.w;
+                // Irradiance arround surface point. 
+                // Increase angle of influence (angle(N,L) -> angle(N,L) + acos(angleOffset)) to smoothen transition regions
+                float3 E = surface.albedo * max(lightingData.angleOffset + dot(-surface.normal, dirToLight),0.0);
```

Comment:
Hm, yes, I grabbed this idea from the [reference paper](http://www.iryoku.com/translucency/downloads/Real-Time-Realistic-Skin-Translucency.pdf), but this will add energy (even though it can be compensated with the scale factor). Maybe a saturate would be more physically correct at the cost of clamping high-energy points to the same value? What do you think?
![image](https://user-images.githubusercontent.com/44426596/147212176-bc06830f-4940-4831-92a3-dbcf4aee3023.png)

Reply:
I think Jeremy has a point and since the albedo is multiplied outside the operation, you must saturate it.  The differences are very small in this example but you can still identify it and it does saturate more than it should.

![image](https://user-images.githubusercontent.com/82479970/147604584-f32e2b8a-58b0-4012-a350-d755a35670e8.png)

On another note - great initiative and results

Reply:
Okay! Done in https://github.com/o3de/o3de/pull/6428/commits/c814be22fd65eacf47ea11d969acc8994780c2eb.